### PR TITLE
Add dilate kwarg to heatmap

### PR DESCRIPTION
**Pull request by fmaussion**
Closes https://github.com/ioam/holoviews/issues/3220

Indeed my test is not very useful, let me know if I should remove it

### CODE REVIEW

File: holoviews/tests/plotting/bokeh/testheatmapplot.py [hunk 1]
```diff
@@ -143,3 +143,7 @@ def test_heatmap_ymarks_list(self):
         for marker, pos in zip(plot.handles['ymarks'], (2, 1)):
             self.assertEqual(marker.location, pos)
             self.assertEqual(marker.dimension, 'width')
+
+    def test_heatmap_dilate(self):
+        # This just ensures that the dilate kwarg is now accepted by heatmap
+        hmap = HeatMap([('A',1, 1), ('B', 2, 2)]).options(dilate=True)
```
Comment (philippjfr) [inline holoviews/tests/plotting/bokeh/testheatmapplot.py]: Small suggestion:

```suggestion
        hmap = HeatMap([('A',1, 1), ('B', 2, 2)]).options(dilate=True)
        plot = bokeh_renderer.get_plot(hmap)
        glyph = plot.handles['glyph']
        self.assertTrue(glyph.dilate) 
```

**Comment by philippjfr** [comment]
Getting some 404s when installing our test env using conda, hopefully it recovers soon.

**Comment by fmaussion** [comment]
> Could you rebase the PR?

Done

**Comment by philippjfr** [comment]
Perfect thanks!

**Pull request merged**